### DISCUSSION TITLE
reshowing a trial if there is an error as opposed to continuing down the timeline

### DISCUSSION THREAD

**Discussion by csavasegal** [jspsych/jsPsych#2236] [Q&A] [answered]
Does anyone have any suggestion for how to reshow the same trial after displaying an alert (as opposed to proceeding to the next one?) Right now I get an alert appropriately, but the timeline continues to the next trial without letting someone fill in the missing input that they got the alert for. I have a sense that I need to add another procedure that potentially loops through the current one, but have not been able to do so successfully. Many thanks in advance!

**Comment by wenbo-lin** [comment, marked as answer]
Have you considered the "required" attribute? 

Example
`<input name="first" type="text" required>`

**Reply by csavasegal** [reply]
Thanks!
